libs/features/cdt/cdt.css

```diff
@@ -49,7 +50,7 @@
   padding: 0 9px;
   width: 10px;
   border-radius: 5px;
-  font-size: var(--type-body-m-size);
+  font-size: 18px;
```

Comment:
Why are you going away from using variables? 🤔

Reply:
As per spec, the font sizes seem fixed and not dependent on the body font size: https://adobe.sharepoint.com/:p:/s/Adobedotcom/Eaw_AFM_SP5HpKZEYktTYdQBkmxRuJ69C0BsakLlSnEXJQ?e=lkcqHv